src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -106,7 +106,7 @@ public void GetAccessControl_FileInfo_AccessControlSections_ReturnsValidObject()
         [Fact]
         public void GetAccessControl_Filestream_InvalidArguments()
         {
-            Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((FileStream)null));
+            Assert.Throws<ArgumentNullException>(() => FileSystemAclExtensions.GetAccessControl((FileStream)null));
```

Comment:
If you like, you could verify the parameter name. That would have caught the bug you fixed by moving the handle innitialization down. eg
`AssertExtensions.Throws<ArgumentNullException>("fileStream", () => FileSystemAclExtensions.GetAccessControl((FileStream)null));`

Reply:
Instead of using "fileStream", we normally use `nameof(fileStream)`. The latter is better because it will catch instances of where parameter names.

Reply:
This is the test, there's no such parameter.

Reply:
The equivalent SetAccessControl test verifies two arguments.  Should this be broken down further? or is `AssertExtensions.Throws<ArgumentNullException("fileStream", () => FileSystemAclExtensions.GetAccessControl((FileStream)null, (FileSecurity)null));`
ok?

Reply:
The two arguments was the correct change. I see you made sure to keep both, thanks @eanova.